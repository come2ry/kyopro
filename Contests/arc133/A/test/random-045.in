9
8 6 10 9 8 6 1 10 9
