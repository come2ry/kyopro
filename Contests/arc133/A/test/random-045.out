6 1 0 9 6 1 1 0 9
